### ISSUE TITLE
Can't pull images

### ISSUE THREAD

**Issue by jayunit100** [kubernetes-csi/external-attacher#120]
After trying to create the external-attacher example YML, I see a perm denied error on the Quay images.... assume theres an easy fix?  Not sure what the one would be though. 

```
[centos@ip-10-0-1-136 demo]$ sudo docker pull  quay.io/k8scsi/mock-plugin
Using default tag: latest
Pulling repository quay.io/k8scsi/mock-plugin
Error: Status 403 trying to pull repository k8scsi/mock-plugin: "{\"error\": \"Permission Denied\"}"
```

(image name change / image deleted) ?

**Comment by pohly** [comment]
The image is called `quay.io/k8scsi/mock-driver`. This needs to be fixed here:
https://github.com/kubernetes-csi/external-attacher/blob/4c7f4801442c44651f398fbc2cdb541f1c29e8ee/deploy/kubernetes/deployment.yaml#L59-L61

@jsafrane: was that image renamed at some point?

**Comment by jayunit100** [comment]
hi again @jsafrane ... images need canary tags to ... added those to the PR ^

**Comment by msau42** [comment]
/close
Fixed by #121
